x/mint/keeper/minter.go

```diff
@@ -1,11 +1,85 @@
 package keeper
 
 import (
+	"math/big"
+	"time"
+
+	"github.com/archway-network/archway/x/mint/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+const Year = 24 * time.Hour * 365
```

Comment:
This gets us `31 536 000` seconds per year. 
Should we replace this with `31 557 600` seconds to accommodate 4 year avg to handle leap years?

Reply:
We should prefer the leap year average... it is better for overall financial math calculations;